mapie/conformity_scores/sets/naive.py

```diff
@@ -0,0 +1,300 @@
+from typing import Optional, Tuple, Union
+
+import numpy as np
+
+from mapie.conformity_scores.classification import BaseClassificationScore
+from mapie.conformity_scores.sets.utils import (
+    check_proba_normalized, get_last_index_included
+)
+from mapie.estimator.classifier import EnsembleClassifier
+
+from mapie._machine_precision import EPSILON
+from mapie._typing import ArrayLike, NDArray
+
+
+class Naive(BaseClassificationScore):
+    """
+    Naive classification non-conformity score method that is based on the
+    cumulative sum of probabilities until the 1-alpha threshold.
+
+    Attributes
+    ----------
+    classes: Optional[ArrayLike]
+        Names of the classes.
+
+    random_state: Optional[Union[int, RandomState]]
+        Pseudo random number generator state.
+
+    quantiles_: ArrayLike of shape (n_alpha)
+        The quantiles estimated from ``get_sets`` method.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def set_external_attributes(
```

Comment:
same as for LAC and TopK

Reply:
it's fine to leave it for future changes

Reply:
If in the future we need to redefine it, then we will rewrite it. However, I think to make the code more concise, we should remove it.